src/lambda/release.js

```diff
@@ -0,0 +1,77 @@
+import fetch from 'node-fetch';
+
+exports.handler = async event => {
+	const { version, url } = await fetchRelease(
+		event.queryStringParameters.repo || 'preactjs/preact'
+	);
+	return {
+		statusCode: 200,
+		body: JSON.stringify({
+			version,
+			url
+		})
+	};
+};
+
+function checkStatus(r) {
+	if (!r.ok) {
+		throw new Error(`${r.status}: Request failed for '${r.url}'`);
+	}
+
+	return r;
+}
+
+function memoize(fn) {
```

Comment:
Not sure how long functions persist and it might be overkill for our tiny function, but should we consider a cache that expires entries? I think https://npm.im/men has this capability

Reply:
That link brings me to an express framework 😅

Reply:
lol whoops! I meant https://npm.im/mem

Reply:
looks like they basically say stuffing in global scope/context can't be relied on cross-request (timeout is 10s):

https://community.netlify.com/t/how-to-pass-values-between-functions/1292

Reply:
Idea: can we set caching headers in our existing netlify headers yaml file? they're the ones calling the function, if we tell them to store it for a period of time I think that prevents the function from being executed again.